CONTRIBUTING.md

```diff
@@ -303,10 +303,32 @@ Various hints for developers
 - *DATALAD_TESTS_SSH*: 
   Skips SSH tests if this flag is **not** set
 - *DATALAD_LOGTRACEBACK*: 
-  Runs TraceBack function with collide set to True, if this flag is set to 'collide' 
+  Runs TraceBack function with collide set to True, if this flag is set to 'collide'.
   This replaces any common prefix between current traceback log and previous invocation with "..."
 - *DATALAD_TESTS_NOTEARDOWN*: 
   Does not execute teardown_package which cleans up temp files and directories created by tests if this flag is set
+- *DATALAD_SPECIAL_REMOTE*:
+  Specifies the name of the datalad [special remote](https://git-annex.branchable.com/special_remotes/).
+- *ARCHIVES_SPECIAL_REMOTE*:
+  Specifies the name of the datalad archive [special remote](https://git-annex.branchable.com/special_remotes/).
+- *DATALAD_USECASSETTE*:
+  Specifies the location of the file to record network transactions by the VCR module.
+  Currently used by when testing custom special remotes
 - *DATALAD_CMD_PROTOCOL*: 
+  Specifies the protocol number used by the Runner to note shell command or python function call times and allows for dry runs. 
+  0 for ExecutionTimeExternalsProtocol, 1 for ExecutionTimeProtocol and 3 for NullProtocol.
+  Any new DATALAD_CMD_PROTOCOL has to implement datalad.support.protocol.ProtocolInterface
```

Comment:
cool, forgot that we have it but it is incorrect values listed -- need to be literals such as 'externals-time' etc -- please fix up... although I also don't see any effect really of using e.g. 'externals-time' -- needs figuring out + fix and a test to sodifiy... feel welcome to tackle or file an issue ;)